### PR TITLE
Add durationToAlertAgain

### PR DESCRIPTION
**Pull request by nwparker**
Adds the ability to use duration to alert again instead of days. Thus allowing more granular control over when to show the alert again

### CODE REVIEW

**Comment by larryaasen** [comment]
Thanks for this PR - this is a good idea. Let me look this over more carefully to find any missing changes.

File: lib/src/upgrade_alert.dart [hunk 1]
```diff
@@ -26,6 +26,9 @@ class _UpgradeBase extends StatefulWidget {
   /// For debugging, display logging statements.
   final bool debugLogging;
 
+  /// Duration until alerting user again after later. Overrides `daysToAlertAgain`
+  final Duration durationToAlertAgain;
```
Comment (larryaasen) [inline lib/src/upgrade_alert.dart:30]: This also needs to be added to classes UpgradeAlert and UpgradeCard.

Comment (nwparker) [reply lib/src/upgrade_alert.dart:30]: Thanks for catching! Done

**Comment by nwparker** [comment]
Thanks for looking it over! I've made the requested changes

**Comment by nwparker** [comment]
bumping @larryaasen 😀

**Comment by nwparker** [comment]
Bumping for the new week

**Comment by larryaasen** [comment]
@nwparker Thanks for this contribution and sorry it took me so long to accept it.

**Pull request merged**